packages/ws-server/src/utils/utils.ts

```diff
@@ -68,35 +69,82 @@ export const getMultipleAddressesEnabled = (): boolean => {
 /**
  * Sends a JSON-RPC response message to the client WebSocket connection.
  * Resets the TTL timer for inactivity on the client connection.
- * @param {any} connection - The WebSocket connection object.
- * @param {any} request - The request object received from the client.
- * @param {any} data - The data to be included in the response.
- * @param {string} tag - A tag used for logging and identifying the message.
- * @param {any} logger - The logger object for logging messages and events.
+ * @param {any} connection - The connection object.
+ * @param {any[]} params - The parameters associated with the request.
+ * @param {any} method - The method associated with the request.
+ * @param {any} response - The response to send to the client.
+ * @param {any} logger - The logger object.
  * @param {string} requestIdPrefix - The prefix for the request ID.
  * @param {string} connectionIdPrefix - The prefix for the connection ID.
  */
 export const sendToClient = (
   connection: any,
-  request: any,
-  data: any,
-  tag: string,
+  params: any,
+  method: any,
+  response: any,
   logger: any,
   requestIdPrefix: string,
   connectionIdPrefix: string,
 ) => {
+  const TX_INFO = params[0];
+  const BLOCK_PARAM = params[1];
+  const TAG = JSON.stringify({ method, txInfo: TX_INFO, block: BLOCK_PARAM });
+
   logger.info(
```

Comment:
should this log be info? or maybe trace or debug?

Reply:
Ah yeah good catch. I'm chaning them to `trace`.